tests/tracking/test_tracking.py

```diff
@@ -370,22 +368,19 @@ def test_with_startrun():
 
 
 def test_parent_create_run(tracking_uri_mock):
-    parent_run = mlflow.start_run()
-    with pytest.raises(Exception, match='To start a nested run'):
-        mlflow.start_run()
-    child_run = mlflow.start_run(nested=True)
-    grand_child_run = mlflow.start_run(nested=True)
+    with mlflow.start_run() as parent_run:
```

Comment:
This should be a no-op refactor: the goal is just to wrap all `start_run` calls in a `with` block so we don't leak active runs